pkg/testutils/kyma.go

```diff
@@ -83,12 +83,13 @@ func GetKyma(ctx context.Context, testClient client.Client, name, namespace stri
 	return kymaInCluster, nil
```

Comment:
@ruanxin, I suggest moving this package to the `/tests/testkit` or `/tests/testutils` (the former one is the convention used by Huskies, so if we adhere to it, it can become a standard in Kyma).

Reply:
For me, putting those utility functions under `pkg` or `tests` is both valid, if you reference to this golang-standards layout, you can find some examples.

for example:
under `pkg` https://github.com/containerd/containerd/tree/main/pkg/testutil
under `test` https://github.com/openshift/origin/tree/master/test/extended/util

but maybe we should rename `/tests` to `/test`, the latter one is the standard name defined in that layout.

regarding the name of `testkit`, IMO, it's not a good name, the functionality and coverage of `kit` are much heavier than `utility` to me, maybe Kuskies need to define some complicated logic to fulfill some test cases, but for us, the function in current package just some lightweight create test data, operator resources in cluster. Another concern, you can also observe in other good structured repo mentioned in that standard layout, most of them use `util` package. 

We can bring this topic to the team, after having decision, create an issue to address this specifically, it's not relevant to this PR, and will make the structure hard to review.

Reply:
The reasoning behind the `testkit` name is the readability (subjective) of the test files:

```go

testkit.CRIsInState(...) /* vs */ utility.CRIsInState(...)

// or 

testkit.CreateNamespace(...) /* vs */ utility.CreateNamespace(...)
```

`testkit` here clearly manifests that a test-related package is involved, and in the case of the `utility`, it is ambiguous if the utility is a part of the application domain or a test one.

So, we need to brainstorm the API for this package and find a proper name and location.

Reply:
regarding this concern, actually from my perperctive, there is no much sense have a name when import test package, that's why I choose this way to [implicit](https://github.com/kyma-project/lifecycle-manager/blob/main/tests/e2e_test/kyma_deletion_test.go#L7) import test package.

And yes, regarding the rest, let's have a dedicate session discuss about it, if you can manage this discussion, it will be great! @ameteiko